android/src/com/frostwire/android/gui/fragments/AboutFragment.java

```diff
@@ -54,40 +53,46 @@ public AboutFragment() {
     public void onActivityCreated(Bundle savedInstanceState) {
         super.onActivityCreated(savedInstanceState);
     }
-    
+
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
         View view = inflater.inflate(R.layout.fragment_about, container, false);
 
+        //Title, build number and changelog setup
         final TextView title = (TextView) view.findViewById(R.id.fragment_about_title);
         final String basicOrPlus = Constants.IS_GOOGLE_PLAY_DISTRIBUTION ? "Basic" : "Plus";
         title.setText("FrostWire " + basicOrPlus + " v" + Constants.FROSTWIRE_VERSION_STRING);
 
         final TextView buildNumber = (TextView) view.findViewById(R.id.fragment_about_build_number);
-        buildNumber.setText("\nbuild " + BuildConfig.VERSION_CODE + " - sdk level " + Build.VERSION.SDK_INT);
+        buildNumber.setText("build " + BuildConfig.VERSION_CODE + Build.VERSION.SDK_INT + ", ");
```

Comment:
hmm, I'd like to keep the "sdk level", I haven't tried this on a phone but it seems to me that this will look like:

`build 9030316`, instead of `build 90303 sdk level 16`, it's so that we can quickly ask users, "hey on what SDK does it say you are on, please go to the About screen"